modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/main.js

```diff
@@ -475,6 +475,21 @@ AUI.add(
 						'destroyPortlet',
 						A.bind(instance._onDestroyPortlet, instance)
 					);
+
+					document.addEventListener('keydown', (event) => {
+						const baseCellEditorPopup = document.querySelector(
+							'.basecelleditor'
+						);
+						if (
+							baseCellEditorPopup &&
+							event.code === 'Enter' &&
+							event.target.height !== 0
```

Comment:
Hey @ramonmsp, I think you can check if keypress was made inside baseCellEditorPopup by using this condition instead of what you did. Take a look:

`baseCellEditorPopup.contains(event.target)`